python/test/CMakeLists.txt

```diff
@@ -319,6 +319,9 @@ ot_pyinstallcheck_test (NearestPointChecker_std)
 if (NLopt_FOUND)
   ot_pyinstallcheck_test (NLopt_std)
 endif ()
+if (Pagmo_FOUND)
+  ot_pyinstallcheck_test (Pagmo_std IGNOREOUT)
```

Comment:
There are results that are printed but not checked with an `assert` statement, are they unimportant?

Reply:
yes, maybe I can add more assert there